client/src/components/ChangePasswordWindow/ChangePasswordWindow.component.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Form, Row, Col, Button } from "react-bootstrap";
+import { toast } from "react-toastify";
+import { logout, changePassword } from "../../redux/actions/authAction";
+import { Fragment } from "react";
+
+const ChangePasswordWindow = () => {
+  const [userCredential, setUserCredential] = useState({
+    currentPassword: "",
+    newPassword: "",
+    confirmPassword: "",
+  });
+
+  const {
+    userInfo: { token, userId },
+  } = useSelector((state) => state.auth);
+  const { errorMsg, success } = useSelector((state) => state.passwordChange);
+
+  const dispatch = useDispatch();
+
+  const handleError = () => {
+    toast("Unable to change password", { type: "error" });
+    return;
```

Comment:
@ranjit977 You don't need to return anything here

Reply:
Alright! Will resolve that. Thanks for pointing it out!